rest/resource-server/src/test/java/org/eclipse/sw360/rest/resourceserver/restdocs/ProjectSpecTest.java

```diff
@@ -2051,6 +2051,53 @@ public void should_document_get_project_report_without_mail_req() throws Excepti
                         )));
     }
 
+    @Test
+    public void should_document_get_project_licenseclearing_report() throws Exception {
```

Comment:
Please update the method name as its not the clearing report but the spreadsheet export.